find-unused-vars.coffee

```diff
@@ -0,0 +1,179 @@
+esprima    = require 'esprima'
+nodeType   = require './node-type'
```

Comment:
I'd rather have this required as `NodeType` to denote that it's an enum type module. 

It makes it clearer that this is a global variable in which place it is used, rather than a local variable. So you immediately look at the top of the file to see where this is defined.